### PR TITLE
Refresh the comment count

### PR DESCRIPTION
**Pull request by piuccio**
## What does this change?

When discussions are enabled on an article, there are two requests for the comment count. One for the icon at the top, one for the discussion block at the bottom.

Now that we're using preact asynchronously, the time difference between the two requests might be higher. In articles with a fast changing comment count this means that the two values are not in sync.

This PR makes sure that the two values are the same after the comments load.
## What is the value of this and can you measure success?

People won't be confused
## Does this affect other platforms - Amp, Apps, etc?

No

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/discussion/discussion-frontend.js [hunk 1]
```diff
@@ -13,6 +13,14 @@ define([
             emitter.once('comment-count', function (value) {
                 if (value === 0) {
                     loader.setState('empty');
+                } else {
+                    // By the time discussion frontent loads, the number of comments
+                    // might have changed. If there are other comment counts element
+                    // in the page refresh their value.
+                    var otherValues = document.getElementsByClassName('js_commentcount_actualvalue');
+                    for (var i = 0, len = otherValues.length; i < len; i += 1) {
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/discussion/discussion-frontend.js]: Don't you want to wrap that in `fastdom.write`?

File: static/src/javascripts/projects/common/views/discussion/comment-count--content.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
 <a href="<%=url%>" data-link-name="Comment count" class="commentcount2 tone-colour">
     <h3 class="commentcount2__heading"><%=icon%> Comments</h3>
-    <span class="commentcount2__value tone-colour"><%=count%></span>
+    <span class="commentcount2__value tone-colour js_commentcount_actualvalue"><%=count%></span>
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/views/discussion/comment-count--content.html]: Everything would be so much simpler if we used `id`s 😭

File: static/src/javascripts/projects/common/modules/discussion/discussion-frontend.js (hunk 1 above)
Comment (piuccio) [inline static/src/javascripts/projects/common/modules/discussion/discussion-frontend.js]: Valid point. I also realized numbers should be formatted. Done

**Comment by regiskuckaertz** [comment]
👍

**Pull request merged**